PhysicsTools/PatAlgos/python/slimming/slimmedGenJets_cfi.py

```diff
@@ -8,6 +8,12 @@
     dropSpecific = cms.bool(False),
 )
 
+slimmedGenJetsFlavourInfos = cms.EDProducer("GenJetFlavourInfoPreserver",
```

Comment:
cfi files are not meant for definitions of many types of module instances.
better create a slimmedGenJetsFlavourInfos_cfi and add this module definition there.

Better yet is to implement a fillDescriptions method and let the framework create this file
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp .